babel.dependencies.config.js

```diff
@@ -1,31 +0,0 @@
-/** @format */
-const config = {
-	// see https://github.com/webpack/webpack/issues/4039#issuecomment-419284940
-	sourceType: 'unambiguous',
-	presets: [
-		[
-			'@babel/env',
-			{
-				modules: false,
-				useBuiltIns: 'entry',
-				corejs: 2,
-				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
-				exclude: [ 'transform-typeof-symbol' ],
-			},
-		],
-	],
-	plugins: [
```

Comment:
`calypso-build` has an additional here `@babel/plugin-proposal-class-properties`: https://github.com/Automattic/wp-calypso/blob/948f23d6a0c7d9243377cb5346997238a160c183/packages/calypso-build/babel/default.js#L16.

Transpiling class properties in dependencies won't hur :heavy_check_mark: